docs/guides/src/main/server/db.adoc

```diff
@@ -73,57 +72,66 @@ Database schema only accounts for Unicode strings in the following special field
+Otherwise, characters are limited to those contained in database encoding, which is often 8-bit. However, for some database systems, you can enable UTF-8 encoding of Unicode characters and use the full Unicode character set in all text fields. For a given database, this choice might result in a shorter string length than the maximum string length supported by 8-bit encodings.
 
-Some of the databases require special settings to database and/or JDBC driver to be able to handle Unicode characters. Please find the settings for your database below. Note that if a database is listed here, it can still work properly provided it handles UTF-8 encoding properly both on the level of database and JDBC driver.
+=== Configuring Unicode support for an Oracle database
 
-Technically, the key criterion for Unicode support for all fields is whether the database allows setting of Unicode character set for VARCHAR and CHAR fields. If yes, there is a high chance that Unicode will be plausible, usually at the expense of field length. If it only supports Unicode in NVARCHAR and NCHAR fields, Unicode support for all text fields is unlikely as the server schema uses VARCHAR and CHAR fields extensively.
+Unicode characters are supported in an Oracle database if the database was created with Unicode support in the VARCHAR and CHAR fields. For example, you configured AL32UTF8 as the database character set. In this case, the JDBC driver requires no special settings.
 
-=== Oracle database
+If the database character set was not created with Unicode support, you need to configure the JDBC driver to support Unicode characters in the special fields. You configure two properties. Note that you can configure these properties as system properties or as connection properties.
 
-Unicode characters are properly handled provided the database was created with Unicode support in VARCHAR and CHAR fields (e.g. by using AL32UTF8 character set as the database character set). No special settings is needed for JDBC driver.
+. Set `oracle.jdbc.defaultNChar` to `true`.
++
+[NOTE]
+====
+Setting this property might negatively impact performance. For more information, see the Oracle JDBC driver configuration documentation.
+====
 
-If the database character set is not Unicode, then to use Unicode characters in the special fields, the JDBC driver needs to be configured with the connection property oracle.jdbc.defaultNChar set to true. It might be wise, though not strictly necessary, to also set the oracle.jdbc.convertNcharLiterals connection property to true. These properties can be set either as system properties or as connection properties. Please note that setting oracle.jdbc.defaultNChar may have negative impact on performance. For details, please refer to Oracle JDBC driver configuration documentation.
+. Optionally, set `oracle.jdbc.convertNcharLiterals` to `true`.
```

Comment:
The new text does not explain the meaning of the properties nor reference to Oracle documentation for their explanation in the rewritten text. The Oracle doc for performance in the `[NOTE]` seems to point only to performance aspects of that property settings.

The correct rephrasing needs to refer to Oracle JDBC driver doc both for explanation of the meaning of the two properties and the performance impact.